### ISSUE TITLE
Dead wiki link in README?

### ISSUE THREAD

**Issue by workingjubilee** [vita-rust/cargo-vita#4]
```quote
Cargo command to work with Sony PlayStation Vita rust project binaries.

For general guidelines see [vita-rust wiki](https://github.com/vita-rust/std/wiki)
```

But it seems to go nowhere?

**Comment by ZetaNumbers** [comment]
Please refer to [our book](https://vita-rust.github.io/book)

**Comment by workingjubilee** [comment]
Oh, all the issues I have filed are not for myself. They are on behalf of someone who was having a great deal of trouble getting the Rust-on-Vita tooling and libraries working after updating to the latest version, because of the extensive difference in workflows and namespaces.

**Comment by ZetaNumbers** [comment]
Fixed in #5

**Comment by pheki** [comment]
@workingjubilee TYSM for the feedback! We've been talking to them on discord to figure out the pain points. It seems like when they started using it, we had basically no tooling and vitasdk-sys was on it's infancy, when I was just blindly linking to all stubs (I'll add more info on the changes and the rationale at vita-rust/vitasdk-sys#20). Since then we've added more docs and it's more organized and centralized, but it seems like discoverability is bad (especially for someone who was familiar the old structure). The docs are now at the [organization root](https://github.com/vita-rust), the [book](https://vita-rust.github.io/book/) and the [examples](https://github.com/vita-rust/examples)

I can relate to the pain though, using unstable dependencies and having to rediscover everything again and again is super annoying.

PS: using `cargo-make` with the same old `Makefile.toml` still works and can still be used, but I wouldn't recommend it
